templates/phaser/packages/client/src/layers/network/contractComponents.ts

```diff
@@ -0,0 +1,24 @@
+/* Autogenerated file. Do not edit manually. */
+
+import { TableId } from "@latticexyz/utils";
+import { defineComponent, Type as RecsType, World } from "@latticexyz/recs";
+
+export function defineContractComponents(world: World) {
+  return {
+    CounterTable: (() => {
```

Comment:
ahh looks like this is either leftover or we're running tsgen to the wrong spot
(this is a dupe of `mud/contractComponents.ts` but outdated)